lib/assert.js

```diff
@@ -67,13 +55,19 @@ function fail(actual, expected, message, operator, stackStartFunction) {
 }
 
 // EXTENSION! allows for well behaved errors defined elsewhere.
-assert.fail = fail;
+assert.fail = function fail(actual, expected, message, operator) {
```

Comment:
It was not about the style here but about the line length. The only other function that was special was `ok` as that was accessed directly before declaration. This change is actually also the reason why I wrote `assert.fail` and not only `fail` as the function was not visible otherwise. I changed changed it in a way that seemed most convenient to me.